requirements.txt

```diff
@@ -23,3 +22,4 @@ pinecone-client
 pinecone-datasets @ git+https://github.com/mrT23/pinecone-datasets.git@main
 loguru==0.7.2
 google-cloud-aiplatform==1.35.0
+litellm==1.7.1
```

Comment:
**Suggestion:** It's a good practice to pin the versions of your dependencies to ensure your project does not break due to updates in the dependencies. Consider pinning the version of `litellm` in your `requirements.txt` file. [best practice]
```suggestion
litellm==1.7.1  # Pin the version to ensure compatibility
```